state/accountsDB.go

```diff
@@ -524,7 +523,8 @@ func (adb *AccountsDB) loadDataTrie(accountHandler baseAccountHandler, mainTrie
 
 	dataTrie, err := mainTrie.Recreate(accountHandler.GetRootHash())
 	if err != nil {
-		return fmt.Errorf("trie was not found for hash, rootHash = %s, err = %w", hex.EncodeToString(accountHandler.GetRootHash()), err)
+		log.Error("trie was not found for hash", "rootHash", accountHandler.GetRootHash(), "err", err)
```

Comment:
log error & return error at the same time? Maybe an error wrap will help?

Reply:
Reverted the changes, the err is now wrapped before returning. Removed the log print.